scripts/check_stubs_version.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+"""Check if stubs version was bumped when .pyi files are changed."""
+
+import subprocess
+import sys
+import tomllib
+from pathlib import Path
+
+_STUBS_ROOT = Path("stubs")
+_STUBS_PYPROJECT = _STUBS_ROOT / "pyproject.toml"
+_ALGOPY_STUBS_PATH = _STUBS_ROOT / "algopy-stubs"
+
+
+def run_git(cmd: list[str]) -> str:
```

Comment:
Minor point but since this is only taking a single arg, make just make it var-args?